pkg/lib/configreader/string.go

```diff
@@ -50,6 +50,7 @@ type StringValidation struct {
 	CastScalar                           bool
 	AllowCortexResources                 bool
 	RequireCortexResources               bool
+	DockerImage                          bool
```

Comment:
I definitely would not have expected you to catch this: do you mind updating `StringPtrValidation()` and `makeStringValValidation()` in `pkg/lib/configreader/string_ptr.go` to include the new field?

Reply:
Yep, updated it. Let me know if I haven't caught something.